tests/testthat/test-FilteredDataset.R

```diff
@@ -152,10 +152,10 @@ testthat::test_that("$get_call returns the filter call of the dataset", {
   filter_call <- shiny::isolate(dataset$get_call())$filter
 
   testthat::expect_equal(
-    deparse1(filter_call),
-    paste0(
+    filter_call,
+    str2lang(paste(
```

Comment:
`quote` here would work perfectly, you can break the lines without concatenating string.